runtime/integration-tests/src/pools/utils/time.rs

```diff
@@ -14,10 +14,20 @@
 
 /// Start date used for timestamps in test-enviornments
 /// Sat Jan 01 2022 00:00:00 GMT+0000
+///
+/// NOTE: In milliseconds!
 pub const START_DATE: u64 = 1640995200u64;
 
+pub mod millisecs {
+	pub const MILLI_SECOND: u64 = 1000u64;
+	pub const MILLI_SECONDS_PER_MINUTE: u64 = 60 * MILLI_SECOND;
+	pub const MILLI_SECONDS_PER_HOUR: u64 = 60 * MILLI_SECONDS_PER_MINUTE;
+	pub const MILLI_SECONDS_PER_DAY: u64 = 24 * MILLI_SECONDS_PER_HOUR;
+	pub const MILLI_SECONDS_PER_YEAR: u64 = 365 * MILLI_SECONDS_PER_DAY;
```

Comment:
Do we care about leap years?

Reply:
Polkadot does by counting 365.25 days per year. So we should probably as well?

Reply:
Since this is used for testing purposes only so far, we might not care about leap years? If we encounter a problem, this will come up in the tests first.